package.json

```diff
@@ -8,9 +8,9 @@
   "private": true,
   "type-check": "tsc --jsx",
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 8080",
```

Comment:
This was changed to `3000` to keep with the defaults

Reply:
3000 conflicts with webpack on onlineweb4 when running locally

Reply:
Should this perhaps be added to the README then?

Reply:
Can revert it and make the change in another pr. But yes, should be included in the README